### DISCUSSION TITLE
Need Help Customizing Crankshaft

### DISCUSSION THREAD

**Discussion by Veltabex** [opencardev/crankshaft#487] [Q&A]
Hey guys I'm trying to customize the existing Crankshaft software to my needs & I have written a small code in .py I would like to run it on the Pi with crankshaft how can I do this ?

Don't mind me asking, I'm new.

**Comment by matt2005** [comment]
What is the python code doing?

**Comment by Veltabex** [comment]
import RPi.GPIO as GPIO # Import Raspberry Pi GPIO library
from time import sleep # Import the sleep function from the time module
GPIO.setwarnings(False) # Ignore warning for now
GPIO.setmode(GPIO.BOARD) # Use physical pin numbering
GPIO.setup(8, GPIO.OUT, initial=GPIO.LOW) # Set pin 8 to be an output pin and set initial value to low (off)
while True: # Run forever
 GPIO.output(8, GPIO.HIGH) # Turn on
 sleep(1) # Sleep for 1 second
 GPIO.output(8, GPIO.LOW) # Turn off
 sleep(1) # Sleep for 1 second

**Comment by Veltabex** [comment]
Also I'm trying to display the crank shaft on my 3.5inch Pi but it won't show its just black screen.

**Reply by matt2005** [reply]
How are you connecting? What is the code doing it appears to be toggling the gpio?

**Reply by Veltabex** [reply]
> How are you connecting? What is the code doing it appears to be toggling the gpio?

Yes one of my friends made it, I have a LCD 3.5 inch it won't display on it.
